mateuszwu/GPP-1
# 22 add scheduler to updater service

This closes #22 


## Overview
Added Quartz Cron Scheduler to trigger NewsUpdater every hour between 7 to 17 from monday to friday


## Testing scenarios
The best way to test this feature is to add `Console.WriteLine` in `UpdaterJob` and comment `NewsUpdater`
 - set your system time to Monday - 15:59 and run `UpdaterService` application
- [x] when 16:00 shows up message you write in `Console.WriteLine` should be printed 
 - set your system time to Sunday - 15:59 and run `UpdaterService` application
- [x] when 16:00 shows up nothing happen
